File: converter-dotnet-core-console-cs/HtmlFileConversion.cs
using ceTe.DynamicPDF.Conversion;
using System;
using System.Collections.Generic;
using System.Text;

namespace converter_dotnet_core_console_cs
{
    public class HtmlFileConversion
    {
        public static void Run(string outputPdf)
        {
            HtmlConversionOptions options = new HtmlConversionOptions(false);

            HtmlConverter htmlConverter = new HtmlConverter(new Uri("http://www.google.com"), options);

            htmlConverter.Convert(Program.GetOutputDocPath(outputPdf));

            Console.WriteLine("HtmlFileConversion Example Executed.");
        }
    }
}

File: converter-dotnet-core-console-vb/HtmlFileConversion.vb
Imports ceTe.DynamicPDF.Conversion

Public Class HtmlFileConversion
    Public Shared Sub Run(outputPdf As String)

        Dim options As HtmlConversionOptions = New HtmlConversionOptions(False)
        Dim htmlConverter As HtmlConverter = New HtmlConverter(New Uri("http://www.google.com"), options)
        htmlConverter.Convert(Program.GetOutputDocPath(outputPdf))
        Console.WriteLine("HtmlFileConversion Example Executed.")

    End Sub
End Class
